lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1875,7 +1875,7 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :default => 10000,
```

Comment:
Would this PR be a good time to add `event_harvest_config.harvest_limits.log_event_data`? Or should we create the setting and switch them up later on?